.github/workflows/env-setup-delete-redis.yml

```diff
@@ -0,0 +1,36 @@
+name: Env Setup - Delete Redis in Openshift
+run-name: Env Setup - Delete Redis in Openshift from ${{ inputs.environment }} using ${{ inputs.gitRef }}
```

Comment:
@andrewsignori-aot @guru-aot @lewischen-aot  @sh16011993 
Do we need the gitref here ?? I couldn't see a reason to use gitref